client/src/components/RatingsReviews.jsx

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
-const RatingsReviews = () => {
-  return null;
-};
-export default RatingsReviews;
\ No newline at end of file
+import axios from 'axios';
+import ReviewList from './Sub_RatingsReviews/ReviewList.jsx';
+
+export default function RatingsReviews({ productID, reviewMetadata, product }) {
+  const sort = 'helpful';
+  // const url = `/api/reviews/?product_id=${productID}&page=1&count=5&sort=${sort}`;
+  const [reviews, setReviews] = React.useState(null);
```

Comment:
it seems unconventional to go with `React.useState` and `React.useEffect` instead of importing useState at the top, like in other files